src/sentry/search/events/fields.py

```diff
@@ -2408,10 +2475,15 @@ def resolve_function(self, function: str, match: Optional[Match[str]] = None) ->
         snql_function = self.function_converter.get(name)
 
         arguments = snql_function.format_as_arguments(name, arguments, self.params)
+        for arg in snql_function.args:
+            if type(arg) == SnQLColumn:
```

Comment:
I can see us potentially subclassing `SnQLColumn` in the near future
```suggestion
            if isinstance(arg, SnQLColumn):
```